CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
```

Comment:
It seems we should release this a 7.12.1 before Firebase 10.18.0 releases with a new requirement on 7.12.x